pipelines/resource_type_test.go

```diff
@@ -131,6 +132,26 @@ var _ = Describe("Configuring a resource type in a pipeline config", func() {
 		})
 	})
 
+	Context("with custom resource types that have params", func() {
+		BeforeEach(func() {
+			flyHelper.ConfigurePipeline(
+				pipelineName,
+				"-c", "fixtures/resource-types-with-params.yml",
+				"-v", "s3_bucket=jghiloni-testflight2",
+				"-v", "s3_regexp=time-resource-(.*).tar.gz",
+				"-v", "s3_region=us-east-2",
+				"-y", "privileged=false",
+			)
+		})
+
+		It("can use a custom resource with parameters", func() {
+			time.Sleep(30 * time.Second)
```

Comment:
Is this sleep still necessary now that the task uses a `docker-image` `image_resource`?

Reply:
Yeah, it is for some reason. The resource goes orange until some time has passed, and adding the sleep back in makes the tests pass.